google-cloud-firestore/src/test/java/com/google/cloud/firestore/LocalFirestoreHelper.java

```diff
@@ -929,22 +929,45 @@ public static String autoId() {
     return fromJsonString(json.replace("'", "\""));
   }
 
-  static class RequestResponseMap
+  /** Map of request/response pairs to be returned when `sendRequest()` is called. */
+  static class RequestResponseStubber
```

Comment:
Nit: This only stubs responses. Maybe ResponseStubber?

It also seems like the comment is no longer applicable.